assets/animations.js

```diff
@@ -25,4 +42,9 @@ function initializeScrollAnimationTrigger(rootEl = document) {
 
 window.addEventListener('DOMContentLoaded', () => initializeScrollAnimationTrigger());
 
-document.addEventListener('shopify:section:load', (event) => initializeScrollAnimationTrigger(event.target));
+if (Shopify.designMode) {
+  document.addEventListener('shopify:section:load', (event) => initializeScrollAnimationTrigger(event, event.target));
+  document.addEventListener('shopify:section:reorder', (event) =>
```

Comment:
@kjellr I added something for the section being re ordered. One thing to note, when I checked, I can only prevent the animation on the section I'm currently trying to re order. Any other section has the animation re running. 

When I checked, `section:load` event wasn't triggered on other sections, so it must be a full page reload. I think it's ok though as long as the section we're currently moving around isn't animating every time we're moving its order. 

[Here is a little video of how it looks](https://screenshot.click/13-18-rxdnv-x24jn.mp4)

Reply:
Hmm ok. I'd definitely prefer to have the other animations stop on reorder too. Can we add a body class with some `!important` rules or something to disable all animations when you're in that mode or something? 😅 

Either way, what we have here is an improvement, so happy to get this merged in and tackle improvements separately. I'll add an approval for now, with the caveat that if we come up with a quick fix to stop all animation on reorder, we should take it.

Reply:
Ok so I can actually do it by making it remove the animation from all present sections that usually do animate. One of the potential issue with this, is that if you're at the top of a page and you haven't fully scrolled down yet and triggered the animations and you're reordering one of your first section on the page, nothing will be animated on further navigation on this page until page reload. 

[Here is a video ](https://screenshot.click/13-24-76sba-d2e80.mp4)cause it's seem hard to explain without visual aid.

Reply:
Ah, nice work! This seems like a pretty good solution. From watching the video, I'm not concerned about the subsequent sections not animating. That seems like a reasonable tradeoff in this case. If you're happy with the dev solution, let's push it up here and test it out.